tap_s3_csv/discover.py

```diff
@@ -27,8 +27,12 @@ def load_metadata(table_spec, schema):
         if table_spec.get('key_properties', []) and field_name in table_spec.get('key_properties', []):
             mdata = metadata.write(
                 mdata, ('properties', field_name), 'inclusion', 'automatic')
+            mdata = metadata.write(
+                mdata, ('properties', field_name), 'source_type', 'string')
         else:
             mdata = metadata.write(
```

Comment:
always do this, no if else

Reply:
do we need to revert setting string as [number, stirng, null]?

Reply:
this provided by singer

Reply:
@woody-feng @yi-varicent meant in conversion.py datatype_schema() we append string as type, we shouldn't need to anymore right? for example discovery should not return [null, 'number', 'string'] for a numeric column?